config/sentinel_config.exs

```diff
@@ -0,0 +1,4 @@
+port 6555
```

Comment:
change the name of the file, calling it `.exs` would confuse other as it's not a valid elixir syntax. Perhaps `.conf` would be better.